spark/sparkstreaming101/src/main/scala/KafkaWordCount.scala

```diff
@@ -24,8 +24,8 @@ object KafkaWordCount {
     val ssc = new StreamingContext(sparkConf, Seconds(10))
 //    ssc.checkpoint("checkpoint")
 
-    Logger.getLogger("org").setLevel(Level.ERROR)
-    Logger.getLogger("akka").setLevel(Level.ERROR)
+      Logger.getLogger("org").setLevel(Level.ERROR)
```

Comment:
Check your indentation or editor , its editor white spaces that you may not have intended